central/enrichment/singleton.go

```diff
@@ -50,6 +51,8 @@ func initialize() {
 		delegatedRegistryConfigDS.Singleton(),
 		connection.ManagerSingleton(),
 		scanwaiter.Singleton(),
+		scanwaiterv2.Singleton(),
+
```

Comment:
Will also need to use `scanwaiterv2` on the 'other end' that provides the updated image to the delegator (assuming thats coming later?):

https://github.com/stackrox/stackrox/blob/514109bc451d3aed3d088eff47283d5c931af342/central/image/service/service_impl.go#L115-L117

Reply:
Yeah the plan is to add delegator and enricher V2 functions first. Then update the service and reprocessor to use the new functions. So I will image service to use `scanWaiterV2` when I come to that.